### ISSUE TITLE
Windows (ARM64) Binary 

### ISSUE THREAD

**Issue by andrewmd5** [bytecodealliance/wasmtime-dotnet#298]
I’ve only just realized that this package does not ship with an ARM64 wasmtime library for Windows - checking up stream it doesn’t appear there is any native release either, despite there being an AARCH64 wheel.

Any chance at getting an experimental cross-compilation build within this package?

Tertiary report [here](https://github.com/betwixt-labs/bebop/issues/322)

**Comment by peterhuene** [comment]
Hi @andrewmd5. We'll need upstream Wasmtime to build the C API package for AARCH64 Windows first; once that artifact is available, I can update `wasmtime-dotnet`'s build to include it.

**Comment by andrewmd5** [comment]
I opened an issue [here](https://github.com/bytecodealliance/wasmtime/issues/8247).

**Comment by andrewmd5** [comment]
I was looking at the workflows and it looks like builds are already produced for this triple?

https://github.com/bytecodealliance/wasmtime/blob/d256e25065d339b922093c28333779a155a61ccb/.github/workflows/main.yml#L381

**Comment by peterhuene** [comment]
Indeed, we appear to `cargo check` it in CI, but traps are non-functional (I would guess we're missing aarch64 unwind information generation for Windows, but I haven't looked at that in quite some time) and tests expect traps to work.

So it looks like this would be blocked until someone has the time to implement support for Windows aarch64.
